.github/workflows/release.yml

```diff
@@ -57,8 +58,10 @@ jobs:
           prerelease: false
           generate_release_notes: true
           files: target/release/macos/harbor.dmg
+
   build-linux:
-    name: Build linux
+    needs: build-mac
+    name: Build Linux
```

Comment:
why does this depend on build-mac now?

Reply:
it was something about the mac build creating the release and then the other builds adding too it... idk I'm never sure about this gh actions stuff